ament_clang_tidy/ament_clang_tidy/main.py

```diff
@@ -252,13 +255,21 @@ def start_subprocess(full_cmd):
             if file_name.endswith(suffix):
                 file_name = file_name[0:-len(suffix)]
         testname = '%s.%s' % (folder_name, file_name)
-        xml = get_xunit_content(report, testname, time.time() - start_time)
+        xml = get_xunit_content(report, testname, elapsed_time)
```

Comment:
nitpick: The changes to capture elapsed time are fine, but they seem orthogonal to the pull request (supporting a sarif output).

Reply:
That was a vestige of the refactoring. During development, the get_sarif_content had the same signature as get_xunit_content, requiring the elapsed time to be passed to both. Later, unused params were dropped from the get_sarif_content interface. 

So, I agree that the result seems to be orthogonal, I think it is a relevant change as the elapsed time isn't specific to the xunit format (and shouldn't be be in that code block. Instead, the elapsed time should be computed before any file output occurs.